src/content/teams/08_moderation.mdx

```diff
@@ -39,8 +39,12 @@ repository](https://github.com/NixOS/moderation).
 
 ## Reporting issues
 
-Contact any of the current team members directly, e.g. via Matrix, or through
-the Contact links.
+The preferred way to contact the team is via private message on the
+[NixOS Discourse](https://discourse.nixos.org/) using the
+`@moderators` group. This allows any member of the team to respond and
```

Comment:
Maybe add a [direct link](https://discourse.nixos.org/g/moderators), though I am not sure if it is visible for all trust levels.

Reply:
Edited.

That's a good point though, you need trust level 1 to send a DM on discourse.

Looking at the stats, 4k users over 15k are trust level 0. Looking at the trust level 1 requirements https://blog.discourse.org/2018/06/understanding-discourse-trust-levels/#trust-level-1-basic , it seem pretty basic and reachable by any active community member though.

People can still fallback on email if they're trust level 0 anyways.